bin/compile

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
 BP_DIR=$(cd $(dirname ${0:-}); cd ..; pwd)
+export BUILDPACK_LOG_FILE=/dev/null
```

Comment:
This was necessary because I was seeing errors like `/tmp/buildpack20200726-1270-8zse1ys/lib/vendor/stdlib_v7.sh: line 138: /dev/fd/3: No such file or directory` on deploy. It appears that `BUILDPACK_LOG_FILE` is being set to `/dev/fd/3` somewhere, but that the nodejs buildpack is not being run with that additional fd. There is probably a better way to address this so that the fd is passed along to the nodejs buildpack.